server/src/main/java/org/elasticsearch/action/admin/cluster/node/stats/NodesStatsRequest.java

```diff
@@ -327,15 +239,15 @@ public void writeTo(StreamOutput out) throws IOException {
         SCRIPT("script"),
         DISCOVERY("discovery"),
         INGEST("ingest"),
-        ADAPTIVE_SELECTION("adaptiveSelection");
+        ADAPTIVE_SELECTION("adaptiveSelection"); // TODO: Should this be removed?
```

Comment:
Can you elaborate on why this might need to be removed?

Reply:
It definitely doesn't need to be removed. I'll remove the `TODO`, which was a reminder for me to look into it.

Reply:
Oh, darn. I do need to change it to `adaptive_selection`, though.